src/app/components/lang-selector/lang-selector.component.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { Component, NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatSelectModule } from '@angular/material/select';
+import { MatSelectHarness } from '@angular/material/select/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { Lang } from '../../models/lang';
+import { LANGUAGES_TOKEN } from '../../tokens/languages.token';
+import { LangSelectorOptionComponent } from '../lang-selector-option/lang-selector-option.component';
+import { LangSelectorComponent } from './lang-selector.component';
+import { LangSelectorHarness } from './lang-selector.harness';
+
+@Component({
+  selector: 'tk-test',
+  template:
+    '<tk-lang-selector [lang]="lang" (langChanged)="langChanged($event)"></tk-lang-selector>'
+})
+class TestComponent {
+  lang: Lang;
+
+  langChanged(value): void {}
```

Comment:
Since you *spyOn* this method, you don't have to provide it here and specify it on the template.